modules/class-simplechart-save.php

```diff
@@ -96,6 +97,14 @@ protected function _do_save_post( $post ) {
 			}
 		}
 
+		// save subtitle separately since we want to be able to save empty strings
+		if ( isset( $_POST['save-chartSubtitle'] ) ) {
+			if ( 'false' !== $_POST['save-chartSubtitle'] ) {
```

Comment:
No, because they're all checking against `empty()` rather than `false` - can't do that for subtitle since it's just a single value that could be an empty string, whereas for `chartMedata` empty fields will be reflected in a json string